backend/service/inviteContact.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-case-declarations */
+import { type User, type Contact } from '@prisma/client';
+import SesService from '../awsSes/SesService';
+import PlanInviteModel from '../models/planInvite';
+
+enum PlanInviteStatus {
+  HAS_INVITED,
+  NOT_INVITED,
+  INVITE_EXPIRED
+}
+
+export default class InviteContactService {
+  static async setupEmailInviteToContacts(planLeader: User, planID: string, contactsArray: Contact[], message: string): Promise<Contact[] | null> {
```

Comment:
rename sendEmailInvitesToContacts or something better, This function is doing too much stuff.